src/parser/monk/mistweaver/modules/spells/EnvelopingMists.js

```diff
@@ -24,15 +23,32 @@ class EnvelopingMists extends Analyzer {
     combatants: Combatants,
   };
 
+  constructor(...args) {
+    super(...args);
+    if (this.selectedCombatant.hasTalent(SPELLS.MIST_WRAP_TALENT.id)) {
+      this.EVM_HEALING_INCREASE=.4;
+    }
+  }
+
+  EVM_HEALING_INCREASE = 0.3;//check for mistwrap
   healingIncrease = 0;
   gustsHealing = 0;
   lastCastTarget = null;
+  countForGusts = false;
+  numberToCount = 0;
 
   on_byPlayer_cast(event) {
     const spellId = event.ability.guid;
-    if (SPELLS.ENVELOPING_MIST.id !== spellId) {
+
+    if (SPELLS.ENVELOPING_MIST.id !== spellId) {//bail early if not the right spell
       return;
     }
+    if (this.combatants.players[event.targetID]) {
+      if (this.combatants.players[event.targetID].hasBuff(SPELLS.ESSENCE_FONT_BUFF.id, event.timestamp, 0, 0) === true) {
+        this.numberToCount++;
+      }
+    }
+    this.numberToCount++;
```

Comment:
Another style item, but we have chosen to use `+= 1` to increment. So just rework this is `this.numberToCount += 1;`

Reply:
Will do Mr.BossMan